README.md

```diff
@@ -1,2 +1,15 @@
 # WagerApp
 An app to aid commitments between parties. More functionalities coming
+
+Scribbling some basic functionalities for user
+1. Create an event with a date (?)
+2. Commit an amount/item to the event
+3. Invite other parties to join the event
+4. View other parties' wagers
+5. Raise a request for other parties to add a wager
+6. Select an event admin (?)
+7. Mark the completion of a user's commitment by themselves or by 3rd party (?)
+8. Approve the completion of a user's commitment
```

Comment:
who will have approval authority?

Reply:
will it be an admin approval or by a vote, say?

Reply:
Admin can be a approver. Until we figure that admin is itself a bottleneck.